### DISCUSSION TITLE
Adding features that use other libraries (nibabel for eg)

### DISCUSSION THREAD

**Discussion by Atif-Anwer** [marksgraham/OCT-Converter#79] [General]
Hi. I was just curious if you were interested in getting pull-requests with features that use other libraries?
I have been working a small project that uses oct-converter and nibabel to convert between Zeiss img format to nifti format. Its working quite well as a standalone script but if its useful (and once i get a go-ahead from my group lead), would you be interested in adding it as an additional feature to oct-converter? Something that can be installed as necessary. 
Its nothing big, but its something that might potentially help others i guess. 
I can share the script before making a pull-request so you can see how it works or if its useful at all for addition.

**Comment by marksgraham** [comment]
Hi @Atif-Anwer ,

Apologies, this message slipped through my net. I think allowing people to save as .nii format would be useful and I think it would be fine to add nibabel as a dependency. I think it would be really nice if it could be implemented as a general method in the `oct_volume` class, so all filetypes could be saved as .nii, but I suppose how feasible that is depends on how much zeiss specific code there is. 

Feel free to share the code!

**Reply by Atif-Anwer** [reply]
No problem at all! That is actually a good idea to be able to cross-convert all file types to nifti. I am currently working on zeiss images so that part is working well, but ill see if i can update the script so that it accepts all file formats supported by oct-converter. Ill get back with a PR after a little while tho!

**Reply by marksgraham** [reply]
Look forward to it!
